scripts/autotracking/archipelago.lua

```diff
@@ -11,6 +11,32 @@ SLOT_DATA = nil
 LOCAL_ITEMS = {}
 GLOBAL_ITEMS = {}
 
+function onClearHandler(slot_data)
+    clear_timer = os.clock()
```

Comment:
I have not tested the suggestion, but if you don't put `local clear_timer` *anywhere*, it's a global, which is bad.
In my testing i made `clear_timer` local to the module, but local to the function would be even better.

```suggestion
    local clear_timer = os.clock()
```

Reply:
Sound good. I didn't pay attention to the namespace behavior.